pylatex/headfoot.py

```diff
@@ -0,0 +1,101 @@
+# -*- coding: utf-8 -*-
+"""
+This module implements the classes that deal with creating headers and footers.
+
+..  :copyright: (c) 2014 by Jelte Fennema.
+    :license: MIT, see License for more details.
+"""
+
+from .base_classes import ContainerCommand, Command
+from .package import Package
+from .utils import NoEscape
+
+
+class PageStyle(ContainerCommand):
+    r"""Allows the creation of new page styles."""
+
+    _latex_name = "fancypagestyle"
+
+    packages = [Package('fancyhdr')]
+
+    def __init__(self, name, data=None, *, header_thickness=0,
+                 footer_thickness=0):
+        r"""
+        Args
+        ----
+        name: str
+            The name of the page style
+        header_thickness: float
+            Value to set for the line under the header
+        footer_thickness: float
+            Value to set for the line over the footer
+        """
+
+        self.name = name
+
+        super().__init__(data=data, name=self.name)
+
+        self.change_thickness(element="header", thickness=header_thickness)
+        self.change_thickness(element="footer", thickness=footer_thickness)
+
+        # Clear the current header and footer
+        self.append(Head(""))
```

Comment:
The empty string represented the position, but I have changed the Head class to work without specifying a position so now the empty string is not required.